@Test(expected=IOException.class) public void resourceUnavailable() throws Exception {
}